### ISSUE TITLE
Carthage build failing on v1.5.1

### ISSUE THREAD

**Issue by andrew-muzz** [RxSwiftCommunity/RxCoreLocation#39]
When running the carthage command to update RxCoreLocation v1.5.1 it fails with:
`framework not found Nimble`

**Comment by bobgodwinx** [comment]
Yes I saw this error on `Xcode 12` but it works perfectly with the `Xcode 11` please see here maybe it might help

https://github.com/Carthage/Carthage/issues/3019


but definitely a fix is needed

**Comment by andrew-muzz** [comment]
Yeah for sure, we do use the script workaround suggested for Xcode 12.
It worked for RxCoreLocation v1.4.1 (just tried again) but not v1.5.1
Also looks to me like RxSwift builds ok with Carthage

**Comment by bobgodwinx** [comment]
I will tray again over the weekend to see if I can fix it. 
But if you find any solution please try to share it so we can fix together.. it will be highly appreciated.

**Comment by keithcml** [comment]
@bobgodwinx can you build with Carthage finally?
